serverkeyapi/api/api.go

```diff
@@ -22,6 +22,12 @@ type ServerKeyInternalAPI interface {
 		request *QueryPublicKeysRequest,
 		response *QueryPublicKeysResponse,
 	) error
+
+	QueryLocalKeys(
```

Comment:
I don't know what local means in this context. We already have a query keys method - `QueryPublicKeys` - why do we need another?